policyengine_us/variables/gov/states/mt/tax/income/deductions/itemized/mt_medical_expense_deduction_joint.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class mt_medical_expense_deduction_joint(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    label = "Montana medical expense deduction when married couples are filing jointly"
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=7",
+        "https://law.justia.com/codes/montana/2022/title-15/chapter-30/part-21/section-15-30-2131/",
+        # MT Code § 15-30-2131 (2022) (1)(g)(i)
+    )
+    unit = USD
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        expense = add(person.tax_unit, period, ["medical_expense"])
+        medical = parameters(period).gov.irs.deductions.itemized.medical
+        # Law does not define Montana AGI as the cap.
+        # Tax form points to page 1, line 14, which is Montana AGI.
+        medical_floor = medical.floor * add(
```

Comment:
is `floor` the right parameter name here, if it's a multiplier? how about something like `agi_share_threshold`?

Reply:
This is a federal parameter and I would not like to change this in this PR since it already has a lot of changes and other state variables rely on it. Filing a separate issue to adjust